package.json

```diff
@@ -3,11 +3,16 @@
   "version": "1.6.2",
   "description": "Parse bytes to human readable sizes (4747) → ('4.75 KB') and vice versa.",
   "main": "dist/index.js",
+  "exports": {
+    "require": "./dist/index.js",
+    "import": "./dist/index.mjs"
+  },
   "types": "typings/index.d.ts",
   "scripts": {
     "test": "jest",
     "build": "babel lib -d dist",
-    "preversion": "npm run build",
+    "module": "echo '\nexport default xbytes;' | cat dist/index.js - > dist/index.mjs",
```

Comment:
This should probably be part of the build task

Reply:
So it's a single `npm run build` for people who also want to use it as a module as opposed to following that with a second `npm run module`

Reply:
Well as you can see `npm run module` is called from `npm run preversion`.
Mind that i think `preversion` is not the right name here. It's supposed to be `prepare` in npm semantics.

Reply:
Documentation:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#devdependencies

Reply:
Also note that ".mjs" extension is carefully chosen here. Using any other extension will let also nodejs fail to load it as es module (i tried .es.js for example and it fails).
See also https://github.com/nodejs/node/issues/34515 for further info.

Reply:
I found a command that is simpler to understand, and added it to "build" target.